src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
But we process most of cmakelists.txt files unless it is excluded using CLR_CROSS_COMPONENTS_BUILD